docs/deployment/linux/red-hat.rst

```diff
@@ -34,7 +34,7 @@ For Red Hat Linux 7, run:
 
 .. code-block:: sh
 
-   sh$ sudo rpm -Uvh https://cdn.crate.io/downloads/yum/7/noarch/crate-release-7.0-1.noarch.rpm
+   sh$ sudo rpm -Uvh https://cdn.crate.io/downloads/yum/7/x86_64/crate-release-7.0-1.x86_64.rpm
```

Comment:
Should we also keep the `noarch` version in a note below, with the hint that that's the variant for CrateDB 4.1?

Reply:
I thought that in 4.1 doc it will be still noarch, but maybe let's add the note 👍

Reply:
Afaik the docs of the howto repo are not versioned like the crate/crate docs are.

Reply:
only docs inside software project repos get versioned